foundation/foundation-mda/src/main/resources/templates/deployment/spark-operator/spark-application-dev-values.yaml.vm

```diff
@@ -1,5 +1,6 @@
 sparkApp:
     spec:
+      image: "${projectName}-spark-worker-docker:latest"
```

Comment:
Same comment as with airflow that it feels like this should be an unnecessary override.  But it might be that you have to change the Tiltfile to also use the full registry. That might be why we've kept this override.  I could accept that this is beyond the scope of the current work, but it would be more correct to align everything on the full registry.

Reply:
Same response as above. Leaving as is for now and can be addressed in separate ticket.